From 3cf4bf5a5382f235b598ee8ee8456ea776482d81 Mon Sep 17 00:00:00 2001
Message-Id: <patch-2.9-3cf4bf5a538-20230202T091827Z-avarab@gmail.com>
In-Reply-To: <cover-0.9-00000000000-20230202T091827Z-avarab@gmail.com>
References: <230131.86357rrtsg.gmgdl@evledraar.gmail.com>
	<cover-0.9-00000000000-20230202T091827Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Wed, 1 Feb 2023 11:46:03 +0100
Subject: [PATCH 2/9] git config docs: document "tar.<format>.{command,remote}"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Since the "tar.<format>.command" and "tar.<format>.remote"
configuration was added in [1] and [2], we have not included it in the
"git-config(1)" docs themselves.

Since we're including "Documentation/config/tar.txt" in
"Documentation/config/git-archive.txt" as of the preceding commit,
let's move this documentation to the former, to be included in the
latter.

This is a move-only change, aside from changing the mention of "`git
archive`" to "linkgit:git-archive[1]", for consistency with other such
mentions.

1. 767cf4579f0 (archive: implement configurable tar filters,
   2011-06-21)
2. 7b97730b764 (upload-archive: allow user to turn off filters,
   2011-06-21)

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Documentation/config/tar.txt  | 18 ++++++++++++++++++
 Documentation/git-archive.txt | 18 ------------------
 2 files changed, 18 insertions(+), 18 deletions(-)

diff --git a/Documentation/config/tar.txt b/Documentation/config/tar.txt
index c68e294bbc5..894c1163bb9 100644
--- a/Documentation/config/tar.txt
+++ b/Documentation/config/tar.txt
@@ -6,3 +6,21 @@ tar.umask::
 	linkgit:git-archive[1] for
 	details. If `--remote` is used then only the configuration of
 	the remote repository takes effect.
+
+tar.<format>.command::
+	This variable specifies a shell command through which the tar
+	output generated by linkgit:git-archive[1] should be piped. The command
+	is executed using the shell with the generated tar file on its
+	standard input, and should produce the final output on its
+	standard output. Any compression-level options will be passed
+	to the command (e.g., `-9`).
++
+The `tar.gz` and `tgz` formats are defined automatically and use the
+magic command `git archive gzip` by default, which invokes an internal
+implementation of gzip.
+
+tar.<format>.remote::
+	If true, enable the format for use by remote clients via
+	linkgit:git-upload-archive[1]. Defaults to false for
+	user-defined formats, but true for the `tar.gz` and `tgz`
+	formats.
diff --git a/Documentation/git-archive.txt b/Documentation/git-archive.txt
index bbb407d4975..268e797f03a 100644
--- a/Documentation/git-archive.txt
+++ b/Documentation/git-archive.txt
@@ -133,24 +133,6 @@ CONFIGURATION
 
 include::config/tar.txt[]
 
-tar.<format>.command::
-	This variable specifies a shell command through which the tar
-	output generated by `git archive` should be piped. The command
-	is executed using the shell with the generated tar file on its
-	standard input, and should produce the final output on its
-	standard output. Any compression-level options will be passed
-	to the command (e.g., `-9`).
-+
-The `tar.gz` and `tgz` formats are defined automatically and use the
-magic command `git archive gzip` by default, which invokes an internal
-implementation of gzip.
-
-tar.<format>.remote::
-	If true, enable the format for use by remote clients via
-	linkgit:git-upload-archive[1]. Defaults to false for
-	user-defined formats, but true for the `tar.gz` and `tgz`
-	formats.
-
 [[ATTRIBUTES]]
 ATTRIBUTES
 ----------
-- 
2.39.1.1392.g63e6d408230

